cmd/eval.go

```diff
@@ -85,7 +89,7 @@ Example: combo eval -r REPLACE_ME=1,2,3 path/to/file
 
 			combinations := combination.NewStream(
 				combination.WithArgs(formatReplacements(replacements)),
-				combination.WithSolveAhead(),
+				combination.WithSolveAhead(useSolvedAhead),
```

Comment:
This is cleaner in this particular spot, but `WithSolveAhead` becomes a slightly confusing argument now. Originally it was just a boolean toggle but now it takes a boolean. I would be okay with just conditionally setting this value in the `eval` command using the old method or renaming the `WithSolveAhead` option function.

Reply:
I'm not sure I entirely agree that `WithSolveAhead(...)` becomes more confusing from an API standpoint. You're essentially just exposing setter-esq functionality as an API option, which is already a fairly prevalent pattern. Maybe we can look into making the API surface more consistent over time if there are any UX problems when interacting with the combinations API?